plugin/keyring_vault/vault_curl.cc

```diff
@@ -149,18 +92,78 @@ std::string Vault_curl::get_error_from_curl(CURLcode curl_code)
   return ss.str();
 }
 
+Secure_string Vault_curl::get_secret_url(const Secure_string &type_of_data) {
+  Secure_ostringstream oss_data;
+
+  DBUG_ASSERT(!vault_credentials_.get_vault_url().empty());
+  oss_data << vault_credentials_.get_vault_url() << "/v1/";
```

Comment:
Just to recheck. Is "/v1/" here a mount point version? If so, shouldn't it be filled in based on secret_mount_point_version?

Reply:
No, `/v1/` is the version of Vault Server API. It is the same `v1` for both `kv` and `kv-v2` secret engines.